cmd/juju/status/status_internal_test.go

```diff
@@ -4833,6 +4855,24 @@ wordpress:logging-dir  logging:logging-directory  logging    subordinate
 	c.Assert(string(output), gc.Equals, expected)
 }
 
+func (s *StatusSuite) TestStatusWithFormatYaml(c *gc.C) {
```

Comment:
@wallyworld are these two what you were after? 1/2

Reply:
These work, thank you. Also need though a tabular version of the test because that's where the decision to use the display name really matters.

Reply:
I've done that on line 4840  by checking machine 1's line looks like:

```plain
1        started  10.0.1.1  snowflake     quantal
```